content/blog/2022/07/psp-migration-script.md

```diff
@@ -0,0 +1,311 @@
+---
+title: Migrate your PSPs to Kubewarden policies!
+authors:
+- José Guilherme Vanz
+date: 2022-07-13
+---
+
+As announced in past blog posts, Kubewarden has 100% coverage of the deprecated,
+and soon to be removed, Kuberernetes PSPs. If everything goes as expected the PSPs will
+be removed in Kubernetes v1.25 due for release on 23rd August 2022.
+
+The Kubewarden team has written a script that leverages the migration tool written
+by [AppVia](https://github.com/appvia/psp-migration), to migrate PSP
+automatically. The tool is capable of reading PSPs YAML and generate the equivalent
+policies in many different policy engines. Therefore, our simple script will migrate
+your PSPs to the equivalent Kuberwarden policies.
+
+The script is available in the [utils repository](https://github.com/kubewarden/utils/blob/main/scripts/psp-to-kubewarden)
+in the Kubewarden GitHub organization. It will download the
+migration tool in the working directory and run it over all your PSPs printing
+the equivalent Kuberwarden policies definition in the standard output. Therefore,
+users can redirect the content to a file or to `kubectl` directly.
+
+The script will migrate the PSPs defined in `kubectl` default context.
+The Kubewarden policies will be printed to stdout. Thus, the users can
+apply it directly or save it for further inspection. Let's take a look at an example:
+
+In a cluster with this PSP applied:
```

Comment:
What does this PSP do? I'd like to reword it as
```suggestion
In a cluster with the PSP for <XYZ> applied:
```

Reply:
It does a lot of things. It block access to host namespaces, block privileged containers, does not allow privilege escalation, forces the drop of all containers capabilities, list the allowed volume types, define the allowed user and groups to be used, controls the supplemental group applied to volumes and force containers to run in a read-only root filesystem.